kubernetes/ftest-kubernetes-assistant/src/test/java/org/arquillian/cube/kubernetes/assistant/HelloWorldKubernetesAssistantTest.java

```diff
@@ -30,8 +31,8 @@ public void should_inject_kubernetes_assistant() {
 
     @Test
     public void should_apply_route_programmatically() throws IOException {
-        kubernetesAssistant.deployApplication("hello-world");
-        Optional<URL> serviceUrl = kubernetesAssistant.getServiceUrl("hello-world");
+        kubernetesAssistant.deployApplication("hello-world");                           // <1>
```

Comment:
Can you check how it's looking in documentation? I have a feeling you should use `kubernetesAssistant.deployApplication("hello-world");           #<!--1-->`. But if `kubernetesAssistant.deployApplication("hello-world");                           // <1>` is working as per expectation then I am fine with this

Reply:
`<!--1-->` is needed if the source is .xml but `<1>` works fine for the java source.